flink-ml-python/pyflink/ml/lib/feature/tests/test_minhashlsh.py

```diff
@@ -0,0 +1,271 @@
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+import functools
+import os
+from typing import List
+
+from pyflink.common import Row, Types
+from pyflink.java_gateway import get_gateway
+from pyflink.ml.core.linalg import Vectors, SparseVectorTypeInfo, DenseVector
+from pyflink.ml.core.wrapper import JavaWithParams
+from pyflink.ml.lib.feature.lsh import MinHashLSH, MinHashLSHModel
+from pyflink.ml.tests.test_utils import PyFlinkMLTestCase
+from pyflink.table import Table
+
+
+class MinHashLSHTest(PyFlinkMLTestCase):
+    def setUp(self):
+        super(MinHashLSHTest, self).setUp()
+        self.data = self.t_env.from_data_stream(
+            self.env.from_collection([
+                (0, Vectors.sparse(6, [0, 1, 2], [1., 1., 1.])),
+                (1, Vectors.sparse(6, [2, 3, 4], [1., 1., 1.])),
+                (2, Vectors.sparse(6, [0, 2, 4], [1., 1., 1.])),
+            ],
+                type_info=Types.ROW_NAMED(
+                    ['id', 'vec'],
+                    [Types.INT(), SparseVectorTypeInfo()])))
+
+        self.expected = [
+            Row([
+                Vectors.dense(1.73046954E8, 1.57275425E8, 6.90717571E8),
+                Vectors.dense(5.02301169E8, 7.967141E8, 4.06089319E8),
+                Vectors.dense(2.83652171E8, 1.97714719E8, 6.04731316E8),
+                Vectors.dense(5.2181506E8, 6.36933726E8, 6.13894128E8),
+                Vectors.dense(3.04301769E8, 1.113672955E9, 6.1388711E8),
+            ]),
+            Row([
+                Vectors.dense(1.73046954E8, 1.57275425E8, 6.7798584E7),
+                Vectors.dense(6.38582806E8, 1.78703694E8, 4.06089319E8),
+                Vectors.dense(6.232638E8, 9.28867E7, 9.92010642E8),
+                Vectors.dense(2.461064E8, 1.12787481E8, 1.92180297E8),
+                Vectors.dense(2.38162496E8, 1.552933319E9, 2.77995137E8),
+            ]),
+            Row([
+                Vectors.dense(1.73046954E8, 1.57275425E8, 6.90717571E8),
+                Vectors.dense(1.453197722E9, 7.967141E8, 4.06089319E8),
+                Vectors.dense(6.232638E8, 1.97714719E8, 6.04731316E8),
+                Vectors.dense(2.461064E8, 1.12787481E8, 1.92180297E8),
+                Vectors.dense(1.224130231E9, 1.113672955E9, 2.77995137E8),
+            ])]
+
+    def test_param(self):
+        lsh = MinHashLSH()
+        self.assertEqual('input', lsh.get_input_col())
```

Comment:
Let's use `lsh.input_col()` here, otherwise lsh.input_col() may not be tested.

Same for other python functions.